scripts/bindings-precommit-hook.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Adds a Python commit hook that only runs if the Python bindings are modified. Extensible to other bindings.